test_conformance/gl/CMakeLists.txt

```diff
@@ -37,42 +45,29 @@ set (GL_SOURCES
     )
 
 if (WIN32)
-    list (APPEND GL_SOURCES ../../test_common/gl/setup_win32.cpp)
+    list (APPEND ${MODULE_NAME}_SOURCES ../../test_common/gl/setup_win32.cpp)
 else(WIN32)
-    list (APPEND GL_SOURCES ../../test_common/gl/setup_x11.cpp)
+    list (APPEND ${MODULE_NAME}_SOURCES ../../test_common/gl/setup_x11.cpp)
 endif(WIN32)
 
 # Compiling GLEW along with the project if the compiler is MINGW.
 # The mingw linker was giving weird errors while linking to glew32.lib generated by
 # MSVC.
 if(MINGW)
-    list (APPEND GL_SOURCES GLEW/glew.c)
+    list (APPEND ${MODULE_NAME}_SOURCES GLEW/glew.c)
     set_source_files_properties(
-        ${GL_SOURCES}
+        ${${MODULE_NAME}_SOURCES}
         COMPILE_FLAGS -DGLEW_STATIC)
     include_directories("./GLEW/")
 endif(MINGW)
 
-if (MSVC)
-    if(CMAKE_CL_64)
-    list(APPEND CLConform_LIBRARIES glew64)
```

Comment:
I know nothing about GLEW really, but both this and the cl12 version of the PR remove all reference of `glew64` , leaving only `glew32`.   Is that intentional?   I.e. will it still work as long as `CLConform_GL_LIBRARIES_DIR` is set appropriately?

Reply:
Here we have some names confusion.
These libraries are  downloaded from the Internet http://glew.sourceforge.net/ .  Check for example glew-2.1.0 . It includes for x64 and x86 in both directories glew32 files. For me it works.
We have just remeber that for windows libraries for x64 or x86 bitness should have names with 32 numbers.